assembly/assembly-wsmaster-war/src/main/java/org/eclipse/che/api/deploy/jsonrpc/CheMajorWebSocketEndpointExecutorServiceProvider.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.api.deploy.jsonrpc;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.slf4j.LoggerFactory.getLogger;
+
+import com.google.common.util.concurrent.ThreadFactoryBuilder;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.LinkedBlockingQueue;
+import java.util.concurrent.SynchronousQueue;
+import java.util.concurrent.ThreadFactory;
+import java.util.concurrent.ThreadPoolExecutor;
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.inject.Provider;
+import javax.inject.Singleton;
+import org.eclipse.che.commons.lang.concurrent.LoggingUncaughtExceptionHandler;
+import org.slf4j.Logger;
+
+/**
+ * {@link java.util.concurrent.ExecutorService} provider used in {@link
```

Comment:
It's how this java doc looks like
![screenshot_20190221_115051](https://user-images.githubusercontent.com/5887312/53159792-3da16300-35cf-11e9-9305-b0e0bfef75a4.png)

```suggestion
/**
 * {@link ExecutorService} provider used in {@link CheMajorWebSocketEndpoint}.
 *
 * Configuration parameters:
 * <ul>
 *   <li>`che.core.jsonrpc.processor_core_pool_size`: corePoolSize of ThreadPoolExecutor</li>
 * <li>`che.core.jsonrpc.processor_max_pool_size`: maximumPoolSize of ThreadPoolExecutor</li>
 * <li>`che.core.jsonrpc.processor_queue_capacity`: if >0 then configures capacity of @{@link
 * java.util.concurrent.LinkedBlockingQueue}, if <=0 then {@link java.util.concurrent.SynchronousQueue} are used.</li>
 * </ul>
 */
```
And with the proposed changes
![screenshot_20190221_115319](https://user-images.githubusercontent.com/5887312/53159844-5a3d9b00-35cf-11e9-9508-b1209f667b79.png)

Reply:
even better exchange \`...\` for `{@literal ...}` or `{@code ...}` for even better looking javadoc :wink: